starfish/pipeline/filter/gaussian_low_pass.py

```diff
@@ -1,24 +1,30 @@
 from functools import partial
+from typing import Union, Tuple
 
 import numpy
 from skimage.filters import gaussian
 
-from starfish.munge import swap
 from ._base import FilterAlgorithmBase
 
 
 class GaussianLowPass(FilterAlgorithmBase):
 
-    def __init__(self, sigma, **kwargs) -> None:
+    def __init__(self, sigma, is_volume: bool=False, verbose=False, **kwargs) -> None:
         """Multi-dimensional low-pass gaussian filter.
 
         Parameters
         ----------
-        sigma : float
+        sigma : Union[float, Tuple[float]]
```

Comment:
type can go into the params?

Reply:
Not sure I understand the question, unless you mean it literally, in which case: It can, Pycharm will check it. I doubt mypy does though, because this would throw an error. 

 I need to figure out a better solution for 3d algorithm support, this duck typing is giving me headaches -- particularly in cases where the data is a 3d volume but the nuclei or dots image is not.

Reply:
Like why isn't it:

`def __init__(self, sigma: Union[float, Tuple[float]], is_volume: bool=False, verbose=False, **kwargs) -> None:`